### DISCUSSION TITLE
Migration fuzziness

### DISCUSSION THREAD

**Discussion by MacelinCane** [twbs/bootstrap#35300] [General]
I'm doing a migration from 4 to 5 and I've hit some strange lingo in the migration docs. 
Can anyone help turn this into an instruction?

In the migration docs it says this...
> .custom-file and .form-file have been replaced by custom styles on top of .form-control.

So what is that meant for us to actually do? Do we eliminate the classes .custom-file and .form-file altogether as long as the parent elements include .form-control?

This "on top of" mention is pretty cryptic.

Thanks

[the remark this answers is not in the document]
**Reply by MacelinCane** [reply]
sorry. :(   Don't know what /CC means.

[the remark this answers is not in the document]
**Reply by JamesVanWaza** [reply]
@MacelinCane It means that he has notified someone else to help you.
